### ISSUE TITLE
Favor modifier if usage weirdness

### ISSUE THREAD

**Issue by tmornini** [rubocop/rubocop#883]
Hey there. Loving Rubocop!

The following code issues this warning:

```
lib/subledger/attribute_validators/d_r_a_validator.rb:45:9: C: Favor modifier if usage when you have a single-line body. Another good alternative is the usage of control flow &&/||.
        if reference !~ URI.regexp
        ^^
```

I understand and appreciate the message, but commenting out that if statement makes Rubocop happy. Seems it _should_ complain about the identical cases above.

Is it possible that Rubocop is smart enough to know that writing them as a single line would violate the line-length limit? If so, kudos, you da' man! :-)

Code:

https://gist.github.com/tmornini/9551852
